src/models/multiBarChart.js

```diff
@@ -334,12 +334,14 @@ nv.models.multiBarChart = function() {
 
     multibar.dispatch.on('elementMouseover.tooltip', function(evt) {
         evt.value = chart.x()(evt.data);
-        evt['series'] = {
-            key: evt.data.key,
-            value: chart.y()(evt.data),
-            color: evt.color
-        };
-        tooltip.data(evt).hidden(false);
+        if (!chart.stacked() || (evt.data.size > 0 || evt.data.size == undefined)) {
```

Comment:
There are some cases where the tooltip is shown with y = 0 (`evt.data.size` is the y value of the column), even if the column is stacked onto another column with y > 0. The effect is not the best.
With this code I solve this problem hiding the tooltip when the graph is stacked and y < 0.
I'll try to reproduce the issue in the updated jsfiddle.